testing-common/src/main/java/io/opentelemetry/instrumentation/testing/InstrumentationTestRunner.java

```diff
@@ -134,21 +134,21 @@ public final <E extends Throwable> void runWithClientSpan(
   }
 
   /**
-   * Runs the provided {@code callback} inside the scope of an CLIENT span with name {@code
+   * Runs the provided {@code callback} inside the scope of an HTTP CLIENT span with name {@code
    * spanName}.
    */
-  public final <T, E extends Throwable> T runWithClientSpan(
+  public final <T, E extends Throwable> T runWithHttpClientSpan(
```

Comment:
I had to make these methods a little more explicit - so that both `span-kind` and `semconv` suppression strategies would work.